src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -193,6 +193,7 @@ ApplicationManagerImpl::ApplicationManagerImpl(
               this, &ApplicationManagerImpl::OnTimerSendTTSGlobalProperties))
     , is_low_voltage_(false)
     , apps_size_(0)
+    , is_registered_in_timeout_(false)
```

Comment:
@ydementieiev rename it according to the meaning please. 
`registered_during_timer_execution` or `...after_timer_expired` or `..before_expiration_timer_start_` , depends on the meaning

Reply:
@AByzhynar  done [c04c115ce](https://github.com/smartdevicelink/sdl_core/commit/c04c115ced5d0ea1dfcc5bb53fc9c01d6d09a998)